student.rb

```diff
@@ -0,0 +1,14 @@
+require_relative 'person'
+
+class Student < Person
+  attr_accessor :classroom
```

Comment:
- [ ] In this case, @classroom is an instance variable that can be accessed and modified within the methods of the Student class, and it is specific to each instance of the class, instance variables are not directly accessible from outside the class. To access them, we typically define getter and setter methods or use the attr_reader, attr_writer, or attr_accessor macros. Please 🙏🏽 remove this line of your code because we don't need to use @classroom outside the class Student.